frontend/src/layout/navigation/MainNavigation.tsx

```diff
@@ -78,38 +78,11 @@ const MenuItem = ({ title, icon, identifier, to, onClick }: MenuItemProps): JSX.
     )
 }
 
-export const MainNavigation = hot(_MainNavigation)
-function _MainNavigation(): JSX.Element {
-    const { user } = useValues(userLogic)
-    const { menuCollapsed, toolbarModalOpen, pinnedDashboardsVisible } = useValues(navigationLogic)
-    const { setMenuCollapsed, collapseMenu, setToolbarModalOpen, setPinnedDashboardsVisible } = useActions(
-        navigationLogic
-    )
-    const navRef = useRef<HTMLDivElement | null>(null)
-    const [canScroll, setCanScroll] = useState(false)
+function PinnedDashboards(): JSX.Element {
```

Comment:
Unsure about the motivation behind this change but it doesn't affect either, so ...

Reply:
Ah, should've been another PR. That MainNavigation component was just too massive with this inside

Reply:
Np, looks good!